web/packages/teleterm/src/deepLinks.ts

```diff
@@ -104,20 +117,41 @@ export function parseDeepLink(rawUrl: string): DeepLinkParseResult {
     };
   }
 
-  if (whatwgURL.pathname !== '/connect_my_computer') {
-    return { status: 'error', reason: 'unsupported-uri' };
-  }
-
-  const { host, hostname, port, username, pathname } = whatwgURL;
-  const url: DeepURL = {
+  // TODO (avatus): remove when authenticate web device case is implemented
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const { host, hostname, port, username, pathname, searchParams } = whatwgURL;
+  const baseUrl: BaseDeepURL = {
     host,
     hostname,
     port,
     // whatwg-url percent-encodes usernames. We decode them here so that the rest of the app doesn't
     // have to do this. https://url.spec.whatwg.org/#set-the-username
     username: decodeURIComponent(username),
-    pathname,
   };
+  switch (pathname) {
+    case `/${Path.ConnectMyComputer}`: {
+      const url: ConnectMyComputerDeepURL = { ...baseUrl, pathname };
+      return { status: 'success', url };
+    }
+    case `/${Path.AuthenticateWebDevice}`: {
+      // const id = searchParams.get('id');
+      // const token = searchParams.get('token');
+      // if (!(id && token)) {
+      //   return { status: 'error', reason: 'unsupported-uri' };
```

Comment:
I feel like this should be 'malformed-url' and then you can attach an error like `new Error('id and token must be included in the deep link for authenticating a web device')`.